src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -66,9 +64,25 @@ public Renderer(IServiceProvider serviceProvider)
         /// <param name="dispatcher">The <see cref="IDispatcher"/> to be for invoking user actions into the <see cref="Renderer"/> context.</param>
         public Renderer(IServiceProvider serviceProvider, IDispatcher dispatcher) : this(serviceProvider)
         {
-            _dispatcher = dispatcher;
+            Dispatcher = dispatcher;
         }
 
+        /// <summary>
+        /// Gets the <see cref="IDispatcher"/>
+        /// </summary>
+        protected IDispatcher Dispatcher { get; }
```

Comment:
Why do we need to expose the dispatcher?

Reply:
Yup, doesn't need to. Undone.